app/src/main/res/menu/menu_feed_fragment.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/menu_item_feed_help"
+        android:icon="?attr/ic_help"
+        android:title="@string/help"
+        app:showAsAction="always" />
```

Comment:
You should tell it to never `showAsAction` here, I don't think it is needed to do it in code
```suggestion
        app:showAsAction="never" />
```

Reply:
Also, the icon is not needed if the item is never shown as action. So you can remove it and also remove the new drawables and `ic_help` attributes

Reply:
It was intended to work this way.

The menu item will be shown as an action when the fragment is not in the main tabs, and when it is, it will be hidden to not clutter up the menus.

My train of thought would be that users will use the feed fragment screen first, as adding it to the main tabs will only be done when the user discovers that feature. I wanted it to be more noticeable at first glance.

---

###### Found yet another bug: the menu is not being set up when returning from another fragment and the feed fragment is in the main tabs (open a video and return to the main screen).

Reply:
Oh that's ok then ;-)